docs/api/wanchain.md

```diff
@@ -52,6 +303,10 @@ Please note, when using [Infra](http://infra.wanchainx.exchange/), only queries
 
 A class instance `variable` holding a `string` which represents the URL to the compiled ERC-721 related smart contract definition file. This file is used when deploying new asset ledgers to the network.
 
+### deployGatewayId
+
+A class instance `variable` holding a `string` which represents an Wanchain address of the [deploy gateway](/#public-addresses).
```

Comment:
A class instance `variable` holding a `string` that represents a Wanchain address of the [deploy gateway](/#public-addresses).